Marlin/Marlin_main.cpp

```diff
@@ -13583,8 +13583,14 @@ void setup() {
       pe_deactivate_magnet(1);
     #endif
   #endif
-}
-
+#if defined (MKS_12864OLED)
+  SET_OUTPUT(LCD_PINS_DC);   
```

Comment:
Also, this is the only place this pin is used. Does this `SET_OUTPUT` do something special that the display needs, even though the pin is never used after this?

Reply:
It was in the patch file for the display.  I can remove it and see if it still works.

Reply:
This pin coincides with `DOGLCD_A0` — they are both assigned to pin 25. During init I guess this pin takes on a different role…?